controllers/userStatus.js

```diff
@@ -65,6 +76,9 @@ const getUserStatus = async (req, res) => {
 const getAllUserStatus = async (req, res) => {
   try {
     const { allUserStatus } = await userStatusModel.getAllUserStatus(req.query);
+    if (req.query.sorted) {
+      getSortedUsers(allUserStatus);
```

Comment:
we are not using the return value. hence I think it will not show the sorted result.